spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BindingServiceConfiguration.java

```diff
@@ -164,12 +169,23 @@ public BinderAwareChannelResolver binderAwareChannelResolver(BindingService bind
 	}
 
 	@Bean
-	@ConditionalOnProperty("spring.cloud.stream.bindings." + ERROR_CHANNEL_NAME + ".destination")
-	public SingleBindingTargetBindable<MessageChannel> errorChannelBindable(
-			@Qualifier(IntegrationContextUtils.ERROR_CHANNEL_BEAN_NAME) PublishSubscribeChannel errorChannel,
+	@ConditionalOnProperty("spring.cloud.stream.bindings." + ERROR_NAME + ".destination")
```

Comment:
Do we need to expose this channel as a bean? It feels like we can do this inside `errorBridgeChannelBindable`.